qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/web/Controller.groovy

```diff
@@ -12,8 +12,4 @@ package life.qbic.portal.qoffer2.web
  */
 class Controller {
 
-    Controller(){
-
-    }
-
```

Comment:
general comment: changes that are not related to the PR title should go into a separate PR :)